src/keybindings.h

```diff
@@ -274,6 +274,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FORMAT_SENDTOCMD8,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD9,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
+	GEANY_KEYS_GOTO_ADD_POSITION,				/**< Keybinding. */
```

Comment:
Currently the key names don't use separators in the last part (`GEANY_KEYS_<GROUP>_<ACTION>`).  Not saying we shouldn't start adding them though, not sure.  @codebrainz @elextr ?